### PR TITLE
Update two new devices in boards.xml

### PR DESCRIPTION
**Pull request by WENLEIDU**
Update EK 261 and FPB 261 in boards.xml

### CODE REVIEW

File: Boards/boards.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+ <?xml version="1.0" encoding="UTF-8"?>
```
Comment (nam-ng) [inline Boards/boards.xml:1]: Q: add space?

File: Boards/boards.xml [hunk 2]
```diff
@@ -37,237 +37,242 @@
...
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX66N_V1.02.bdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX66N_V1.01.bcdf"/>
   </board>
   <board id="TargetBoardRX671" version="1.02" description="Target Board for RX671 Board Description File" keys="RX">
-    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e0eeb869eb1c6203a513aaefdf7fe27b1a4494fd/Boards/TargetBoardRX671_V1.02.bdf"/>
-    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e0eeb869eb1c6203a513aaefdf7fe27b1a4494fd/Boards/TargetBoardRX671_V1.01.bcdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX671_V1.02.bdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX671_V1.01.bcdf"/>
   </board>
-</boards>
```
Comment (nam-ng) [inline Boards/boards.xml:273]: Q: this line is removed?

File: Boards/boards.xml (hunk 1 above)
Comment (WENLEIDU) [reply Boards/boards.xml]: Space removed, please check

File: Boards/boards.xml (hunk 2 above)
Comment (WENLEIDU) [reply Boards/boards.xml:273]: Sorry for missing this, made mistake when copy paste, added.

**Review by nam-ng — APPROVED** [review]

**Pull request merged**